src/core/stream/index.ts

```diff
@@ -193,15 +193,16 @@ export default function Stream({
   const stream$ = observableCombineLatest(
     openMediaSource(mediaElement),
     fetchManifest(url)
-  ).pipe(mergeMap(([ mediaSource, manifest ]) => {
+  ).pipe(mergeMap(([ mediaSource, { manifest, manifestFetchingDuration } ]) => {
     const loadStream = StreamLoader({ // Behold!
       mediaElement,
       manifest,
+      manifestFetchingDuration,
```

Comment:
Why not regrouping these two under a `manifestInfos` name or something. With both the manifest and the requestTime)